cmd/gateway/validation.go

```diff
@@ -13,7 +13,7 @@ import (
 
 const (
 	// nolint:lll
-	// Regex from: https://github.com/kubernetes-sigs/gateway-api/blob/v0.7.1/apis/v1beta1/shared_types.go#L495
+	// Regex from: https://github.com/kubernetes-sigs/gateway-api/blob/v0.8.0/apis/v1beta1/shared_types.go#L495
```

Comment:
```suggestion
	// Regex from: https://github.com/kubernetes-sigs/gateway-api/blob/v0.8.0/apis/v1beta1/shared_types.go#L551
```
they moved the regex in .8

Reply:
Whoops, I must've clicked the old link and thought it didn't move :)